apps/src/lab2/views/components/AiTutor2Chat.tsx

```diff
@@ -47,38 +48,9 @@ const modelParameters: ModelParameters = {
 
 // Some pre-canned chat buttons.
 const defaultChatButtonData: ChatButtonData[] = [
```

Comment:
The content for this list in unchanged, it just now lives in apps/src/aiTutor/suggestedPrompts.ts